test/map/NearCachedMapTest.js

```diff
@@ -77,33 +77,24 @@ describe('NearCachedMapTest', function () {
             it('second get should hit', async function () {
                 await map1.get('key0');
                 const val = await map1.get('key0');
-                const stats = await getNearCacheStats(map1);
                 expect(val).to.equal('val0');
-                expect(stats.missCount).to.equal(1);
-                expect(stats.entryCount).to.equal(1);
-                expect(stats.hitCount).to.equal(1);
+                expectStats(map, 1, 1, 1);
```

Comment:
The tests failed due to this line. This should be `map1` instead of `map`. Can you change that?